satpy/etc/readers/modis_l2.yaml

```diff
@@ -86,3 +93,856 @@ datasets:
     file_type: mod35_hdf
     coordinates: [longitude, latitude]
 
+##########################
+#Datasets in file mod06_l2
+##########################
+
+# file contents: https://atmosphere-imager.gsfc.nasa.gov/sites/default/files/ModAtmo/MOD06_L2_CDL_fs.txt
+
+  brightness_temperature:
+    name: brightness_bemperature
```

Comment:
Typo here, this should be 'brightness_temperature'

Reply:
```suggestion
    name: brightness_temperature
```